scripts/gatsby-plugin-checker/index.js

```diff
@@ -64,6 +64,60 @@ const filterNotNotified = (packages, plugins) => {
   return packages.filter(pkg => notified.indexOf(pkg.name) < 0)
 }
 
+const removePackagesWithoutRepository = packages =>
+  packages.filter(p => hasRepository(p))
```

Comment:
There is no need to create another function as it's only used once.

```suggestion
  packages.filter(package => !!package.links.repository)
```